package.json

```diff
@@ -28,7 +28,8 @@
     "@polymer/polymer": "^3.0.0",
     "@types/chai": "^4.1.7",
     "@types/mocha": "^5.2.6",
-    "@types/sinon": "^7.0.9",
+    "@types/sinon": "^7.0.11",
```

Comment:
Can;t tell from here, but these look like they are dependencies. They should be dev-dependencies